portal-api/api-tests/v1/gef/external-api.api-test.js

```diff
@@ -82,15 +82,24 @@ describe(baseUrl, () => {
       expect(body).toEqual([
         {
           status: 422,
+          errMsg: {},
           errCode: 'ERROR',
           errRef: 'postcode',
         },
       ]);
     });
 
-    it('Returns a not found address if the postcode was invalid', async () => {
-      const { status } = await as(aMaker).get(`${baseUrl}/address/A1`);
+    it('Returns a invalid postcode if the postcode was invalid', async () => {
```

Comment:
Can we add `500` API test assertion please.

Reply:
Added test `returns a 422 response if backend returns 500`